cdn.md

```diff
@@ -1,31 +1,41 @@
+When serving reader-facing content from an S3 bucket, you should still have Fastly in front, and ideally the S3 bucket would be private. Fastly have [a guide](https://docs.fastly.com/en/guides/amazon-s3#using-an-amazon-s3-private-bucket) on how to add the required authentication headers.
 
-## Configuration
+## Fastly
 
-Fastly is a highly programmable CDN, but we advise making use of the smallest subset of the features necessary to achieve your goal.
+### DNS Settings
+
+The DNS record for your public-facing domain should be a CNAME to Fastly's Guardian-specific hostname:
+
+* **`dualstack.guardian.map.fastly.net.`** - preferred, enables [IPv6 support](https://docs.fastly.com/en/guides/ipv6-support)
+* `guardian.map.fastly.net.` _(only if IPv6 support is somehow causing issues, which would be unusual)_
+
+### Minimise Fastly VCL configuration
+
+Fastly is highly programmable through its VCL configuration language, but VCL can be difficult to work with and we advise making use of the smallest subset of the features necessary to achieve your goal.
 
 A lot can be achieved with minimal Fastly configuration, and careful use of cache-control, surrogate-control and surrogate-key headers served by your application. This has the advantage that most of the caching logic is co-located with the rest of your application.
 
-If this is insufficient, the next step is making use of VCL snippets, which can be easily edited in the Fastly console and provide a useful way of providing a little extra functionality.
+If this is insufficient, the next step is making use of [VCL Snippets](https://docs.fastly.com/en/guides/using-regular-vcl-snippets), which can be easily edited in the Fastly console and provide a useful way of providing a little extra functionality. You can try-out snippets of Fastly VCL functionality with https://fiddle.fastly.dev/ .
 
-If you find that your VCL snippets are becoming quite large, you should consider switching to custom VCL, which should be versioned in Github and deployed using riff-raff.
+If you find that your VCL snippets are becoming large, you should consider switching to [custom VCL](https://docs.fastly.com/en/guides/uploading-custom-vcl), which should be versioned in Github, tested in CI and deployed using riff-raff, as in
+https://github.com/guardian/fastly-edge-cache.
 
-## Shielding
+### Shielding
 When using Fastly it is recommended that you enable [shielding](https://docs.fastly.com/en/guides/shielding#caveats-of-shielding) as it will improve cache performance. Bear in mind that when using shielding any VCL may be executed more than once - see [caveats of shielding](https://docs.fastly.com/en/guides/shielding#caveats-of-shielding). 
 
 We suggest using the London-LCY point of presence as your shield.
```

Comment:
Incidentally, https://github.com/guardian/fastly-edge-cache/issues/986 may be of interest, regarding the choice of LCY as shield.